package.json

```diff
@@ -17,7 +17,7 @@
     "start": "microbundle-crl watch --no-compress --format modern,cjs --jsxFragment React.Fragment",
     "storybook": "storybook dev -p 6006",
     "storybook:build": "storybook build",
-    "storybook:serve:start": "pm2 delete storybook -s 2> /dev/null || true && pm2 start ./serve-storybook.js --name storybook",
+    "storybook:serve:start": "pm2 start ./serve-storybook.js --name storybook",
```

Comment:
Adjusted script to not attempt to delete storybook on `storybook:serve:start`. The `storybook:serve:stop` script will delete storybook.

The`test:playwright` script should build, start, test then stop, so we shouldn't need to run the start and stop scripts separately.